lsp/package.json

```diff
@@ -13,6 +13,7 @@
     "@github/copilot-language-server": "^1.322.0"
   },
   "devDependencies": {
+    "@biomejs/biome": "^2.0.5",
```

Comment:
Let's keep this PR scoped to the changes in `index.ts`. We could have a follow up to add linting/formatting. Would you mind restoring this file and `pnpm-lock.yaml` back to what is on `main`?

```sh
git co main -- lsp/package.json
git co main -- lsp/pnpm-lock.yaml
```